core/proto/keytransparency_v1_proto/keytransparency_v1_admin_proto.proto

```diff
@@ -0,0 +1,108 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/google/keytransparency/core/proto/keytransparency_v1_proto";
```

Comment:
Should 'admin' appear in the package name?

Reply:
Good question. What do you think?  Trillian puts all the APIs in the same `trillian` package and separates out the different services with different `service` definitions and files which is the pattern I've tried to follow here.